### PR TITLE
Infer the language instead of requiring the user to input it

### PR DESCRIPTION
**Pull request by Naatan**
https://www.pivotaltracker.com/story/show/173569440

### CODE REVIEW

File: internal/runners/packages/list.go [hunk 1]
```diff
@@ -139,7 +139,7 @@ func fetchCheckpoint(commit *strfmt.UUID) (model.Checkpoint, *failures.Failure)
 		return nil, model.FailNoData.New(locale.T("package_no_data"))
 	}
 
-	return model.FilterCheckpointPackages(checkpoint), fail
+	return checkpoint, fail
```
Comment (MDrakos) [inline internal/runners/packages/list.go:142]: This change doesn't seem to be related to importing packages. This will also include requirements that are not packages in the output of `state packages`

Comment (Naatan) [reply internal/runners/packages/list.go:142]: It's related since otherwise I cannot filter out the language. I'll address the second point.
